docs/mechanical/resourcesM.md

```diff
@@ -0,0 +1,12 @@
+Resources
+=========
+
+Here is a list of the resources we recommend to getting started designing killer autonomous submarines and boats:
```

Comment:
You're mixing the markdown syntax (`.md`) and rst syntax (`.rst`) here and in several other places... pick one. Be sure to actually build and preview the docs before pushing them up, here you would see that they are misformated